PeerioTechnologies/peerio-mobile
# feat: iOS native action to share

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/epic/5912/mobile-share-with-to-send-files-to-peerio
https://app.clubhouse.io/peerio/story/5956/ios-build-share-with-peerio-extension
#### Testing instructions  
1. Open peerio-mobile/ios/peeriomobile.xcodeproj in XCode
2. Select the project 'peerioshare' and run it. 
<img width="322" alt="screen shot 2018-08-07 at 15 13 07" src="https://user-images.githubusercontent.com/3017559/43775179-761f29ae-9a54-11e8-8b75-15701ad749c5.png">

3. Choose Photos for images / Files for PDFs.
4. Open the context menu for a file and 'Upload to Peerio' should be listed as an option (if it's not listed, it should be under the 'More' section)
5. If you are logged in, after the loading animation stops you are taken in the Files tab and see an upload modal preview. 

Nothing happens if you're not logged in.
We currently support loading only one file.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
